contracts/schema/Schema.sol

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity >=0.7.0 <0.9.0;
+/**
+ * @title Schema
+ * @dev Schema contract
+ */
+contract SchemaRegistry {
+
+    struct Schema {     
+        address issuer;
+        string name;
+        bytes32 hash;  
+        bytes body;
+        uint256 timestamp;
+    }
+
+    mapping(string => bytes32) nameHash;
+    mapping(bytes32 => Schema) hashSchema;
+    
+    function getHashFromBytes(bytes memory schemaBody) private pure returns (bytes32) {
+        return keccak256(schemaBody);
+    }
+
+    /**
+     * @dev save is function to store schema
+     * @param schemaName - name of the schema
+     * @param schemaBody - json schema representation in bytes 
+     */
+    function save(string calldata schemaName, bytes calldata schemaBody) public {
```

Comment:
It would be good to know gas estimates for some data. For example, how much gas it costs to save 100, 200, 500 bytes schemaBody and how much $ with current ~2500 ETH/USD price it is.
So it would be good to emulate the function calls and write down the results somewhere

Reply:
agree, don't think it will run on ethereum anyway, but agree that we need to measure gas cost. we will create a separate task for this.